src/unix/aix.c

```diff
@@ -64,11 +64,19 @@
 #define RDWR_BUF_SIZE   4096
 #define EQ(a,b)         (strcmp(a,b) == 0)
 
+static uv_mutex_t mutex;
```

Comment:
General observation: can you call this `process_title_mutex` or something like that?  Just `mutex` is a bit too generic.  Likewise for `once`.